.eslintrc

```diff
@@ -104,7 +104,6 @@
     "no-implicit-coercion": 2, // disallow the type conversions with shorter notations
     "no-implicit-globals": 2, // disallow var and named functions in global scope
     "no-implied-eval": 2, // disallow use of eval()-like methods
-    "no-invalid-this": 2, // disallow this keywords outside of classes or class-like objects
```

Comment:
You can enable this rule again, but at the same time u need to change those lines:
https://github.com/Andarist/react-collapse/blob/badc37ddc3f377f84c1291effbe8e08acf42ad5f/.eslintrc#L10
https://github.com/Andarist/react-collapse/blob/badc37ddc3f377f84c1291effbe8e08acf42ad5f/.eslintrc#L107

plus ofc `npm install --save-dev eslint-plugin-babel`